router/router.go

```diff
@@ -33,6 +33,7 @@ func GetRouter() *mux.Router {
 	router.HandleFunc("/databases", controllers.GetDatabases).Methods("GET")
 	router.HandleFunc("/schemas", controllers.GetSchemas).Methods("GET")
 	router.HandleFunc("/tables", controllers.GetTables).Methods("GET")
+	router.HandleFunc("/health", controllers.GetHealthStatus).Methods("GET")
```

Comment:
Change endpoint from `/health` to `/_health`, we should default to an endpoint starting with `_` for “fixed” (not dynamic) endpoints